install_config/persistent_storage/dynamically_provisioning_pvs.adoc

```diff
@@ -347,6 +354,47 @@ storage that are registered with it. Trident itself is configured separately.
 <1> For more information about installing Trident with {product-title}, see the link:https://github.com/NetApp/trident[Trident documentation].
 <2> For more information about supported parameters, see the link:https://github.com/NetApp/trident#storage-attributes[storage attributes] section of the Trident documentation.
 
+[[change-default-storage-class]]
+== Changing the default StorageClass
+Use the following process to change the default StorageClass:
```

Comment:
You mention it in the note box you added above, but it might be good to mention why this would be needed: If you are using GCE and AWS.